packages/web/src/scss/components/Dropdown/README.md

```diff
@@ -1,30 +1,140 @@
+      </svg>
+      <span>More links</span>
+    </a>
+    <a href="#" class="d-flex mb-400">
+      <svg width="24" height="24" aria-hidden="true" class="mr-400">
+        <use xlink:href="/icons/svg/sprite.svg#profile" />
+      </svg>
+      <span>Profile</span>
+    </a>
+    <a href="#" class="d-flex">
+      <svg width="24" height="24" aria-hidden="true" class="mr-400">
+        <use xlink:href="/icons/svg/sprite.svg#help" />
+      </svg>
+      <span>Help</span>
+    </a>
+  </div>
+</div>
+```
+
+## Advanced usage
+
 ```html
-<div class="Dropdown is-open">
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#info" />
-    </svg>
-    <span>Information</span>
-  </a>
-  </a>
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#link" />
-    </svg>
-    <span>More links</span>
-  </a>
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#profile" />
-    </svg>
-    <span>Profile</span>
-  </a>
-  <a href="#" class="d-flex">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#help" />
-    </svg>
-    <span>Help</span>
+<div class="dropdown">
+  <button
+    data-toggle="dropdown"
+    data-target="#DropdownDemo2"
+    data-reference="parent"
+    data-placement="top-start"
+    data-offset="5"
+    data-padding="5"
+  >
+    toggle
+  </button>
+  <div class="Dropdown is-open" id="DropdownDemo2">
+    <a href="#" class="d-flex mb-400">
+      <svg width="24" height="24" aria-hidden="true" class="mr-400">
+        <use xlink:href="/icons/svg/sprite.svg#info" />
+      </svg>
+      <span>Information</span>
+    </a>
+    </a>
+    <a href="#" class="d-flex mb-400">
+      <svg width="24" height="24" aria-hidden="true" class="mr-400">
+        <use xlink:href="/icons/svg/sprite.svg#link" />
+      </svg>
+      <span>More links</span>
+    </a>
+    <a href="#" class="d-flex mb-400">
+      <svg width="24" height="24" aria-hidden="true" class="mr-400">
+        <use xlink:href="/icons/svg/sprite.svg#profile" />
+      </svg>
+      <span>Profile</span>
+    </a>
+    <a href="#" class="d-flex">
+      <svg width="24" height="24" aria-hidden="true" class="mr-400">
+        <use xlink:href="/icons/svg/sprite.svg#help" />
+      </svg>
+      <span>Help</span>
+    </a>
+  </div>
 </div>
 ```
+
+## Placement
+
+Default placement of dropdown content is set as `bottom-start`, but you can set it to different position such as `top-start`, `top-end`, `right-start`, `right-end`, `bottom-end`, `left-start` or `left-end`.
+
+## JavaScript
+
+There are two options here. Use the trigger element as an anchor or wrap the menu together with the trigger into a `.dropdown` class.
```

Comment:
As agreed in person, the latter option is becoming `DrowdownWrapper` for now. We would change it to some kind of a helper class later on (not in this PR).